torch/nn/modules/normalization.py

```diff
@@ -141,20 +141,21 @@ class LayerNorm(Module):
         >>> output = m(input)
     """
     __constants__ = ['normalized_shape', 'eps', 'elementwise_affine']
-    normalized_shape: _shape_t
+    normalized_shape: Sequence[int] 
```

Comment:
This should be `Tuple[int, ...]`, because there's an explicit coercion on line 153 and the tuple is of variable length.

Reply:
Also, `_shape_t` should be changed from `List[int]` to `Sequence[int]` on line 83 for consistency it looks like.